src/mobx/stores/Plugins.ts

```diff
@@ -211,7 +227,7 @@ export default class Plugins implements Store, Persistent<Data> {
             loaded.onUnload?.();
             this.plugins.set(ns, {
                 ...plugin,
-                enabled: true,
+                enabled: false,
```

Comment:
true :trollface:

Reply:
false :trollface:

Reply:
null :trollface:

Reply:
NaN :trollface:

Reply:
> true :trollface:

I might as well ask; the plugin is never set to disabled anywhere, which is why I made this change. Unless I am awfully blind and it happens elsewhere.

Reply:
No this change is correct, I put the troll face because I fucked up lol